lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -621,15 +627,21 @@ def write(chunk)
           severity = compute_severity(
             entry_resource.type, record, entry_common_labels)
 
+          # Extract "trace" as LogEntry field according to specified config
+          trace = record[@trace_key]
```

Comment:
I renamed it to `trace_resource` as per the definition of the field in LogEntry, it's actually a resourceful path for the trace, so something with the format "projects/[PROJECT-ID]/traces/[TRACE-ID]" for Stackdriver Trace, or something else for traces from other tracing systems. See: https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry